advanced/Templates/pihole-FTL.service

```diff
@@ -21,8 +52,6 @@ start() {
   else
     # Touch files to ensure they exist (create if non-existing, preserve if existing)
     mkdir -pm 0755 /run/pihole
-    [ ! -f /run/pihole-FTL.pid ] && install -m 644 -o pihole -g pihole /dev/null /run/pihole-FTL.pid
-    [ ! -f /run/pihole-FTL.port ] && install -m 644 -o pihole -g pihole /dev/null /run/pihole-FTL.port
```

Comment:
Are you sure that FTL can create those files as `pihole` user? `/run` isn't world-writable! As the PID file path can be changed, probably the function needs to be split, to have one for obtaining the PID file path, and a dedicated one for obtaining the PID itself. Then the intended PID file path can be pre-created.

Reply:
I think FTL is starting with root power, creating those files, [does some other stuff] and later dropping privileges. But this is best answered by @DL6ER

Reply:
No, FTL is aleady as unprivileged user. It used to be started as `root` and then dropping after binding the ports on Docker, but this was recently changed to all-unprileged, too.

Reply:
Ok, then I'll basically revert the above change and use the detected PID file path.